ghost/core/content/plugins/infobox/card.css

```
@@ -0,0 +1,97 @@
+.infobox-card {
+  display: flex;
+  gap: 16px;
+  padding: 20px 24px;
+  margin: 24px 0;
+  border-radius: 10px;
+  border-left: 5px solid;
+  font-family: 'Inter', system-ui, -apple-system, sans-serif;
```

Stylelint (17.13.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)